invenio_records_lom/cli.py

```diff
@@ -10,15 +10,51 @@
 import click
 from flask.cli import with_appcontext
 from invenio_access.permissions import system_identity
+from invenio_search import current_search
+from invenio_search.errors import IndexAlreadyExistsError
 
 from .fixtures import publish_fake_records
 from .proxies import current_records_lom
 
 
 @click.group()
+@click.version_option("0.1", prog_name="invenio-records-lom cli")
 def lom():
     """CLI-group for "invenio lom" commands."""
-    pass
+
+
+@lom.command("add-indices")
+@with_appcontext
+def add_indices():
```

Comment:
This should not be in this PR. Create a separate PR for that CLI command.

Reply:
Removed.